R/arima.R

```diff
@@ -449,7 +448,11 @@ specials_arima <- new_specials(
 #' where \eqn{\mu} is the mean of \eqn{(1-B)^d y_t}{(1-B)ᵈ yₜ} and \eqn{c = \mu(1-\phi_1 - \cdots - \phi_p )}{c = μ(1-φ₁ - ⋯ - φₚ )}.
 #'
 #' @section Specials:
+#' 
+#' The _specials_ define the space over which `ARIMA` will search for the model that best fits the data. If the RHS of `formula` is left blank, the default search space is given by `pdq() + PDQ()`: that is, a model with candidate seasonal and nonseasonal terms, but no exogenous regressors (other than a mean). Note that a seasonal model requires at least 2 full seasons' worth of data; if this is not available, `ARIMA` will revert to a nonseasonal model with a warning.
```

Comment:
I would leave the "mean" parenthesis out of this, it is better considered as a constant (as it can impose a trend or higher order curve in the presence of differences).